src/dog.sol

```diff
@@ -173,8 +173,15 @@ contract Dog {
 
             dart = min(art, mul(room, WAD) / rate / milk.chop);
 
+            uint256 mart = mul(art - dart, rate);
```

Comment:
This actually should be renamed by something else. `art` is used for normalized debt, in this case as we are multiplying by `rate` it is not actually an `art`.